recipe/meta.yaml

```diff
@@ -373,10 +381,6 @@ outputs:
         - pykerberos >=1.1.13
         - requests-kerberos >=0.10.0
         - thrift_sasl >=0.2.0
-        # temporary dependencies because of bad build of thrift_sasl
-        - thrift >=0.13
-        - pure-sasl >=0.6.2
-        - six >=1.13.0
```

Comment:
This isn't fixed yet.  I'm waiting on https://github.com/conda-forge/thrift_sasl-feedstock/pull/16

Reply:
There's build 1 of the latest `thrift_sasl` still isn't quite right but doesn't cause trouble with `pip check`.  I'll still try to fix it in https://github.com/conda-forge/thrift_sasl-feedstock/pull/17